src/Version.php

```diff
@@ -32,11 +32,46 @@ public static function isStable()
     }
 
     /**
-     * Compares a version to Bolt's version.
+     * Compares a semantic version against Bolt's version, given a specified
+     * comparison operator.
      *
-     * Note: Be sure to include the `.z` number in the version given, as omitting it can give inconsistent results.
-     * For example, if the current version is `3.3.0`, `compare('3.3', '>=')` returns false. In reality you want
-     * the check to return true for this case, and `compare('3.3.0', '>=')` _does_ return true.
+     * @see http://semver.org/ For an exmplaination on semantic versioning.
+     *
+     * Note: Be sure to include the `.z` number in the version given, as
+     * omitting it can give inconsistent results. See below at the results in
+     * Tables 1 & Table 2 for more detail.
+     *
+     * Table 1: `const VERSION = '3.2.0'
+     * +--------------+----+--------+
+     * | $version     |    | Result |
+     * +--------------+----+--------+
+     * | 3.2          | >= | false  |
+     * | 3.2.0        | >= | true   |
+     * +--------------+----+--------+
+     *
+     * Note: Depending on the stability of the VERSION constant, the output of
+     * this may give, initially, unexpected results.
+     *
+     * The two tables below show `Version::compare() calls, compared version,
+     * operator, and their result. Firstly versus Bolt 3.2.0, a stable version
+     * (Table 1). Secondly against Bolt 3.2.0-beta9, a beta (Table 2).
+     *
+     * Table 2: `const VERSION = '3.2.0 beta 9'`
+     * +--------------+----+--------+
+     * | $version     |    | Result |
+     * +--------------+----+--------+
+     * | 3.2          | >= | false  |
+     * | 3.2.0-beta8  | >= | false  |
+     * | 3.2.0-beta9  | >= | true   |
+     * | 3.2.0        | >= | true   |
+     * +--------------+----+--------+
+     *
+     * Note: Pre-release versions, such as 3.2.0-beta, are considered lower
+     * than their final release counterparts (like 2.3.0). As you may notice,
+     * the difference being that '3.2.0-beta9' is considered LOWER than both
+     * Bolt '3.2.0', and '3.2'.
```

Comment:
I would just keep this paragraph and remove the two paragraphs and the table above it. They are all saying the same thing.